site.yml.sample

```diff
@@ -463,3 +467,28 @@
       delegate_to: "{{ groups['mons'][0] }}"
       run_once: true
       when: not ceph_status.failed
+
+- hosts: ceph-grafana
+  become: true
+  tasks:
+    - import_role:
+        name: ceph-defaults
+      tags: ['ceph_update_config']
+      when: dashboard_enabled
+    - import_role:
+        name: ceph-prometheus
+      when: dashboard_enabled
+    - import_role:
+        name: ceph-grafana
+      when: dashboard_enabled
+
+- hosts: mgrs[0]
```

Comment:
There is no mgrs[0] by default anymore, now we collocate mon/mgr by default. So 3 mons = 3 mgrs